.github/workflows/ci_freebsd.yml

```diff
@@ -25,5 +25,4 @@ jobs:
         with:
           name: all_test_output
           path: |
-            release/Testing/Temporary
-            build/Testing/Temporary
+            output
```

Comment:
Should we consider changing the "if: always()" here to "if: failure()"?

Reply:
you're saying we only care about artifacts on failures? i think that makes sense

Reply:
Yeah but I now realize it won't help. It's the rsync in the run step that's failing, not the upload step. [This](https://github.com/vmactions/freebsd-vm/blob/main/run.sh#L164) is probably our problem :\